webpack-widget.js

```diff
@@ -102,37 +127,142 @@ const getLegacyWidgetBuildConfig = (config = {}) => {
 		})
 	}
 
+	const devDemo = 'demo_dev.json'
```

Comment:
Breaking Change / New Feature: arbitrarily naming a dev demo.json file is no longer possible, however if you use `demo_dev.json` it will automatically be loaded.

Reply:
This doesn't quite work - it will write a `demo_dev.json` file into the dev server, but the dev server is hard-coded to use `demo.json`.

Either the webpack config needs to write the contents of `src/demo_dev.json` into `demo.json` on the dev server, or the express code needs to check for `demo_dev.json` and use that if it exists instead of `demo.json`.